lib/exq/redis/job_stat.ex

```diff
@@ -145,4 +186,30 @@ defmodule Exq.Redis.JobStat do
         val
     end
   end
+
+  def get_redis_commands(namespace, node_id, started_at, master_pid, queues, work_table, poll_timeout) do
```

Comment:
Maybe name this something more descriptive, above methods that are similar are named something like <usage>_commands